package.json

```diff
@@ -1,11 +1,13 @@
 {
   "dependencies": {
+    "@hint/formatter-html": "^1.0.1",
     "algoliasearch": "^3.30.0",
     "applicationinsights": "^1.0.3",
     "body-parser": "^1.18.3",
+    "consolidate": "^0.15.1",
```

Comment:
We should check if `consolidate` is needed now that you have removed `handlebars`.